src/test/ui/traits/wf-trait-object-maybe-bound.rs

```diff
@@ -0,0 +1,15 @@
+// compile-pass
+
+// Test that `dyn ... + ?Sized + ...` is okay (though `?Sized` has no effect in trait objects).
```

Comment:
Open a clippy issue to complain about `?Sized` bounds in trait objects

Reply:
See https://github.com/rust-lang/rust/pull/59445#discussion_r280169613; My preference (changed my mind based on discussion) is to just ban `?Sized` again until such time that we have a stronger motivation to lift the ban.

Reply:
I'm not changing again with the potential to keep reversing on the issue. Niko gave his blessing to allowing them, so I'll do what @oli-obk said and open a clippy issue.

Reply:
I did not see the other discussion. Of course we don't want random churn. So before touching it we should discuss further in the comment thread @Centril linked to.

Reply:
As per the other discussion (which can't be commented on anymore), please ban `?Sized` in `dyn` again.